src/bundle/Resources/public/js/scripts/admin.notifications.modal.js

```
@@ -179,9 +179,11 @@
 
     const notificationsTable = modal.querySelector(SELECTOR_TABLE);
     currentPageLink = notificationsTable.dataset.notifications;
+    let interval = Number.parseInt(notificationsTable.dataset.notificationsCountInterval);
+    if (!interval) { interval = INTERVAL; }
```

Fallback on hardcoded `INTERVAL` in case of `NaN` or `0`.